src/emailservice/Dockerfile

```diff
@@ -1,27 +1,52 @@
+FROM python:2.7-alpine3.8 as base
```

Comment:
It would be great if we could keep it python3.

Reply:
👍 Will move back to Python 3

Reply:
Have moved everything to python 3